app/views/pages/hiring_fundraiser.html.haml

```diff
@@ -0,0 +1,137 @@
+        Over the last 3 years, we have raised ~£500k in support from organisations including #{link_to 'Mozilla', 'https://www.mozilla.org/en-GB/', target: '_blank', rel: :noopener}, #{link_to 'Packt Publishing', 'https://www.packtpub.com/about', target: '_blank', rel: :noopener}, #{link_to 'GoBridge', 'https://exercism.org/partners/gobridge', target: '_blank', rel: :noopener}, and the #{link_to 'University of Chicago', 'https://www.uchicago.edu/', target: '_blank', rel: :noopener}.
+        Your role is to take that ad-hoc work, formalise it into a coherent strategy, and then execute on that to deliver £500k of new support in the next 12 months.
+        If the initial work is successful, we will reinvest some of the funds into growing a fundraising team around you.
+        Although most of your role will be autonomous, but will have close access to the CEO to strategise and explore opportunities as needed.
+      %p.text-p-xlarge.mb-12
+        The key areas of your work will be:
+
+      %ul.text-p-xlarge.mb-40
+        %li Work with the CEO to develop an overarching fundraising strategy.
+        %li Identify, approach, and close deals with potential sponsors.
+        %li Identify partnership opportunities and work with the CEO to close them.
+        %li Identify and apply for grants.
+
+      %h2.text-h2.mb-12 Is this job right for you?
+      %p.text-p-xlarge.mb-12
+        The right person for this role will be someone with the experience to devise and execute a fundraising strategy, who cares about free lifelong education (preferably programming or STEM), and who shares our values.
+      %p.text-p-xlarge.mb-12
+        Maybe you're relatively early in your career, looking to take a step up into a position of more responsibility.
+        Maybe you're coming towards the end of the career and looking to give back to a not-for-profit.
+        Or maybe you're somewhere in between. All are fine by us.
+        Similarly, we don't require any specific academic qualifications.
+        Our main criteria for hiring will be the belief that you will be able to execute in the role.
+        For that, we need to see some track-record of fundraising and/or sales and your ability to think and plan strategically.
+      %p.text-p-xlarge.mb-12
+        At Exercism, we have a shared commitment to kindness, honesty, and curiosity, and strive to build a working environment where everyone can learn, improve and do their best work.
+        We believe that everyone has the right to feel happy, safe and supported at work, and strive to ensure that is the case at Exercism.
+        We celebrate diversity, and do better work because of the different perspectives and insights that different experiences bring.
+        It's important to us that you share these values and want to be part of a team living them out.
+      %p.text-p-xlarge.mb-40
+        Finally, we want someone who can get passionate about Exercism.
```

Comment:
I don't think we need someone who is currently passionate about Exercism, but to my mind we definitely want someone who cares about the things we care about and the type of things that we're doing, so that they're likely to become passionate about Exercism.